defaults/main.yml

```diff
@@ -9,7 +9,8 @@ create_volume: true
 
 docker_service_prefix: docker.
 method: pull
-service_pre_command: "/usr/bin/docker pull {{ docker_image }}"
+service_pre_command:
```

Comment:
It would be better to just comment it out with the default value. This way the if condition will be "cleaner".